carma/launch/carma_src.launch.py

```diff
@@ -123,14 +123,23 @@ def generate_launch_description():
         actions=[
             PushRosNamespace(EnvironmentVariable('CARMA_ENV_NS', default_value='environment')),
             IncludeLaunchDescription(
-                PythonLaunchDescriptionSource([ThisLaunchFileDir(), '/environment.launch.py'])
+                PythonLaunchDescriptionSource([ThisLaunchFileDir(), '/environment.launch.py']),
+                launch_arguments = { 
+                    'subsystem_controller_param_file' : [vehicle_config_dir, '/SubsystemControllerParams.yaml'],
```

Comment:
While it could, I'm not sure I see a particular value. Its not likely we would want to specify this file in a way where the folder wouldn't be sufficient